Pratiksha/Javascript/Assignment9/Q5.html

```diff
@@ -1,25 +1,24 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Q5</title>
-</head>
-<body>
-     <script>
-         var str =prompt("Enter String");
-         var words = str.split("")
-         console.log(words.length);
-         let counter = 0;
-         for( let i = 0; i<str.length; i++)
-         {
-             if (str.charAt(i) == '')
-             {
-                 counter++
-             }
-         }
-         console.log(counter);
-     </script>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Word and Whitespace</title>
+  </head>
+  <body>
+    <script>
+      var str = prompt("Enter String");
+      var words = str.split("");
+      console.log(words.length);
+      let counter = 0;
+      for (let i = 0; i < str.length; i++) {
+        if (str.charAt(i) == "") {
+          counter++;
+        }
+      }
+      console.log(counter);
+    </script>
+  </body>
+</html>
```

Comment:
not getting the correct output.Also the output is displayed on console and not on web page